templates/newsletter_form.php

```diff
@@ -0,0 +1,45 @@
+<?php 
+	$user = wp_get_current_user();
+	$user_meta = get_user_meta($user->ID, 'community-meta-fields', true);
+	$user_language = is_array($user_meta) && sizeof($user_meta) > 0 && isset($user_meta['languages']) && sizeof($user_meta['languages']) > 0 ? $user_meta['languages'][0] : '';
+	$user_country = is_array($user_meta) && sizeof($user_meta) > 0 && isset($user_meta['country']) ? $user_meta['country'] : '';
+?>
+<h2 class="profile__form-title">Get Updates</h2>
+<p>Subscribe to our newsletter and join Mozillians all around the world and learn about impactful opportunities to support Mozilla’s mission.</p>
+<input type="hidden" id="fmt" name="fmt" value="H">
+<input type="hidden" id="newsletters" name="newsletters" value="about-mozilla">
+<div class="row profile__newsletter__container">
+	<div class="col-lg-4 profile__newsletter__fields">
+		<label class="newsletter__label" for="newsletter-email">Email</label>
+		<input class="newsletter__input" aria-label="Enter your e-mail" aria-required="true" type="email" id="newsletter-email" name="newsletter-email" value="<?php echo (isset($user->user_email) && strlen($user->user_email) > 0 ? $user->user_email : '')?>">
+		<span class="newsletter__error">Invalid email.</span>
+	</div>
+	<div class="col-lg-4 profile__newsletter__fields">
+		<label class="newsletter__label" for="newsletter-country">Country or Region</label>
+		<select class="newsletter__dropdown" id="newsletter-country" name="country">
```

Comment:
You will need to change the name of this field, this is causing the profile not to save the the users country.

Reply:
I also did not get any error when I did not select a country.  Are country and language required for the newsletter API?

Reply:
Language and country are both optional for the Newsletter API.

Reply:
Alright, so I think you'll just need to change the name of the country field.  Right now country isn't saving on the profile

Reply:
Thanks! Will do!